source/common/upstream/original_dst_cluster.cc

```diff
@@ -62,7 +62,8 @@ OriginalDstCluster::LoadBalancer::chooseHost(const LoadBalancerContext* context)
             Network::Utility::copyInternetAddressAndPort(*dst_ip));
         // Create a host we can use immediately.
         host.reset(new HostImpl(info_, info_->name() + dst_addr.asString(), std::move(host_ip_port),
-                                envoy::api::v2::Metadata::default_instance(), 1, ""));
+                                envoy::api::v2::Metadata::default_instance(), 1,
+                                envoy::api::v2::Locality()));
```

Comment:
default_instance() ? (Same in other places)

Reply:
Not technically needed, since `HostDescriptionImpl` makes a copy, but I guess there is some minor performance gain.